backend/app/models.py

```diff
@@ -20,3 +21,20 @@ def to_dict(self):
     def __repr__(self):
         """Method to return a string representation of the model"""
         return f"Item('{self.name}', '{self.description}')"
+
+class User(db.Model):
+    """Users model"""
+    __tablename__ = 'users'
+
+    id = db.Column(db.Integer, primary_key=True)
+    username = db.Column(db.String(80), unique=True, nullable=False)
```

Comment:
Always include created_at, updated_at and deleted_at timestamps while defining a model. 

- [ ] Include timestamps

Reply:
Is deleted_at timestamp necessary? I think it wouldn't be meaningful after we delete an instance

Reply:
It's necessary if you want to support soft deletes.

Reply:
> Is deleted_at timestamp necessary? I think it wouldn't be meaningful after we delete an instance

Would be nice if you refer to the use of deleted_at in soft deleting.

Reply:
Ok I'll read about soft deletes thank you